sdk/python/kfp/components/structures_test.py

```diff
@@ -871,13 +865,10 @@ def test_concat_placeholder(self):
                 container=structures.ContainerSpecImplementation(
                     image='alpine',
                     command=[
-                        'sh', '-c', 'echo "$0"',
-                        placeholders.ConcatPlaceholder(items=[
-                            placeholders.InputValuePlaceholder(
-                                input_name='input1'),
-                            placeholders.InputValuePlaceholder(
-                                input_name='input2')
-                        ])
+                        'sh',
+                        '-c',
+                        'echo "$0"',
+                        "{{$.inputs.parameters['input1']}}+{{$.inputs.parameters['input2']}}",
```

Comment:
It looks like we had a bug with `ContactPlaceholder` that it added the "+" when it shouldn't be.
Do we have a serialization test that verifies it's no longer the case?

Reply:
I think [this test](https://github.com/kubeflow/pipelines/blob/7adcc831035473f7aa35ab661bab8c944b61754f/sdk/python/kfp/components/placeholders_test.py#L143-L164) is sufficient. Can you let me know if you think there is some aspect this doesn't capture?

Reply:
Sorry, I think the error was previously on the deserialization path--which no longer exists after this change. So we should be good.

Looking at the diff here, previously `"{{$.inputs.parameters['input1']}}+{{$.inputs.parameters['input2']}}"` was parsed into 
```
placeholders.ConcatPlaceholder(items=[
                            placeholders.InputValuePlaceholder(
                                input_name='input1'),
                            placeholders.InputValuePlaceholder(
                                input_name='input2')
                        ])
```

which serializes into 

```
{"Concat": ["{{$.inputs.parameters['input1']}}", "{{$.inputs.parameters['input2']}}"]}
``` 

and resolved at runtime as 

```
"{{$.inputs.parameters['input1']}}{{$.inputs.parameters['input2']}}"
```

(no plus sign in the middle)